backend.js

```diff
@@ -52,6 +52,7 @@ module.exports = {
 				svg: 'always',
 			},
 		],
+		'import/newline-after-import': ['error'],
```

Comment:
we should add the following, therefore adding comments right after import statements will be considered invalid.

more info: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/newline-after-import.md

```suggestion
		'import/newline-after-import': ['error', { considerComments: true }],
```

Reply:
Good suggestion, added it 👌